static/app/views/monitors/components/overviewTimeline/monitorEnvironmentLabel.tsx

```diff
@@ -1,26 +1,54 @@
 import styled from '@emotion/styled';
 
 import {Tooltip} from 'sentry/components/tooltip';
+import {IconFix, IconMute} from 'sentry/icons';
+import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
+import type {ColorOrAlias} from 'sentry/utils/theme';
 import {
   type Monitor,
   type MonitorEnvironment,
   MonitorStatus,
 } from 'sentry/views/monitors/types';
-import {statusIconColorMap} from 'sentry/views/monitors/utils/constants';
+import {
+  statusIconColorMap,
+  type StatusNotice,
```

Comment:
i think we have a `types` file, shouild it go in there?

Reply:
yea was questioning whether it warranted going in a separate types file since the type felt so specific but i'll move it there for consistency